test/e2e/mocha-reporter.json

```diff
@@ -0,0 +1,6 @@
+{
+	"reporterEnabled": "mocha-teamcity-reporter, @automattic/mocha-debug-reporter",
+	"automatticMochaDebugReporterReporterOptions": {
```

Comment:
`ReporterReporter` is not a typo.

Doc (https://www.npmjs.com/package/mocha-multi-reporters#configuring-reporters) says "Specify each reporter's configuration using its camel-cased name, followed by `reporterOptions`, as key. For scoped reporters such as example @myorg/custom, remove all special characters.": `automatticMochaDebugReporter` + `ReporterOptions`